index.js

```diff
@@ -39,7 +39,14 @@ var cli = new Liftoff({
   processTitle: makeTitle('gulp', process.argv.slice(2)),
   completions: completion,
   extensions: interpret.jsVariants,
-  v8flags: v8flags,
+  v8flags: function(callback) {
+    v8flags(function(err, flags) {
+      if (!err) {
+        flags.push('--experimental_modules');
```

Comment:
This isn't a v8 flag but a Node.js specific flag, so it isn't discovered by `v8flags`. This flag is required in order to use ESM modules before Node.js 13.2.0.

Reply:
And before 12.17.0 (https://nodejs.org/en/blog/release/v12.17.0/), for those not aware.

Reply:
I submitted a pull request, making `v8flags` detect non-v8 flags like this: gulpjs/v8flags#55

Reply:
I removed this workaround and required `v8flags ^3.2.0` which includes my change instead.